CMakeLists.txt

```diff
@@ -47,6 +47,9 @@ project(SFML)
 # include the configuration file
 include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/Config.cmake)
 
+# we use the gnu install dirs for our install paths
```

Comment:
Don't like this comment. First of all, it should be "GNU", not "gnu", plus it kinda makes it sound like these paths are set in stone and not configurable. Instead, we should somehow lead the reader to [this link](https://cmake.org/cmake/help/git-master/module/GNUInstallDirs.html) and say that all these paths are configurable.

Reply:
Probably just adding the link is good enough? Don't think we need to go into a detailed description about cmake features?

Reply:
Yeah, maybe add a link and say "search for CMAKE_INSTALL_* for customizing installation directories". But maybe even link should be enough

Reply:
Have added the link and mentioned that they can be set by the user (and made GNU uppercase :P)